reviews/models/mixins.py

```diff
@@ -12,6 +13,18 @@
 from reviews import workflow
 from reviews.exceptions import InvalidTriggerError
 
+from website import settings
+
+from osf.models import NotificationDigest
+from osf.models import OSFUser
+
+
+from website import mails
+from website.notifications.emails import get_user_subscriptions, get_node_lineage
+from website.notifications import utils
+
+reviews_email = signal('reviews_email')
```

Comment:
It looks like existing code uses `blinker.Namespace()`, instead of creating signals in the global namespace. We should probably do that too.